src/index.html

```diff
@@ -9,8 +9,44 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="fonts" href="./fonts/Avenir-Book.ttf">
+    <link rel="fonts" href="./fonts/Avenir-Heavy.ttf">
+    <link rel="fonts" href="./fonts/Avenir.ttc">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=" "
+      method="post"
+      data-qa="big"
+    >
+      <div class="search__bar">
+        <label>
+          <input
+            type="text"
+            name="search"
+            class="search__bar-input"
+            placeholder="Try &quot;Los Angeles&quot"
+            data-qa="keypress"
+          >
+        </label>
+      </div>
+    </form>
+    <form
+      action=" "
+      method="post"
+      data-qa="small"
+    >
+      <div class="search__bar--small">
+        <label>
+          <input
+            type="text"
+            name="search"
+            class="search__bar-input--small"
```

Comment:
when you name search bar small, expected big somewhere :)

Reply:
https://github.com/mate-academy/layout_search-bar-airbnb/pull/2884#discussion_r1111573639